src/components/TableHeadCell.js

```diff
@@ -68,6 +68,8 @@ class TableHeadCell extends React.Component {
     hint: PropTypes.string,
     /** Column displayed in print */
     print: PropTypes.bool.isRequired,
+    /** Tooltip label to fall back to */
+    label: PropTypes.string,
```

Comment:
This doesn't look like it's being used anywhere. Was this a holdover from the previous changes?

Reply:
Forgot to change it with "column". Updated.